// apps/FrontEndServer/app/config.js
var url = require("url");
try {
    var defaultConfigValues = require("../../../../../config.js")
} catch (e) {
    var defaultConfigValues = {}
}
var hostString = defaultConfigValues.clientLocation || process.env.npm_config_adl_host_url || "http://serviceadvisor.deere.com",
    offlineString = defaultConfigValues.offlineLocation || process.env.npm_config_adl_offline_url || "http://serviceadvisorlocal.deere.com:3074",
    redirectWhenOnline = defaultConfigValues.redirectWhenOnline || process.env.npm_config_adl_redirectWhenOnline || "redirect",
    hostUrl = url.parse(hostString),
    offlineUrl = url.parse(offlineString);
exports.host = {
    host: hostUrl.hostname,
    offlinehost: offlineUrl.hostname,
    onlineUrl: hostString,
    offlineUrl: offlineString,
    whenOnline: {
        target: offlineString
    },






    whenOffline: {
        target: offlineString
    }
}, exports.server = {
    ip: "127.0.0.1",
    port: offlineUrl.port || 80
}, exports.secureServer = {
    ip: "127.0.0.1",
    port: offlineUrl.port ? offlineUrl.port + 1 : 443
}, exports.name = "front-end-server";
